ubuntudesign/developer.ubuntu.com
# Update site to Vanilla 1.6.5

# Done

- Updated site to Vanilla 1.6.5, which includes:
  - Header and sidebar navigation
  - Base templates
  - Card templates
  - Grid templates
- Refactored scss files and minimised bespoke styling as much as possible

# QA
- Pull code and run `./run`
- Navigate to http://0.0.0.0:8015/
- Compare with https://developer.ubuntu.com/ and check everything looks/runs okay (especially the nav) and it's all Vanilla-fied

# Issues
- Fixes vanilla-framework/vanilla-framework#1466
